CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.2.6 April 17, 2023
```

Comment:
```suggestion
## 1.2.0 April 17, 2023
```

Usually with semver, you bump to the next minor version and set the patch level to 0.

Reply:
done. ready for re-review